### PR TITLE
feat: Fill body

### PR DESCRIPTION
**Pull request by 97baek**
- added my info

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,9 +1,19 @@
...
-  <body></body>
+  <body>
+    <h1>Welcome To My Blog</h1>
+    <div>
+      <ul>
+        <li><h3>이름: 백상흔</h3></li>
+        <li><h3>생년월일: 1997.07.07</h3></li>
+        <li><h3>거주지: 경기도 남양주시</h3></li>
+        <li><a href="https://github.com/97baek">github</a></li>
+      </ul>
```
Comment (imhaeunk) [inline index.html]: Could you plz use ol?

File: index.html [hunk 2]
```diff
@@ -1,9 +1,19 @@
...
+<!DOCTYPE html>
 <html>
   <head>
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>My first HTML page with git</title>
   </head>
-  <body></body>
```
Comment (imhaeunk) [inline index.html:8]: Could you use just one body tag :)

**Review by imhaeunk — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (97baek) [reply index.html]: I used ol.

File: index.html [hunk 3]
```diff
@@ -1,9 +1,19 @@
...
+  <body>
+    <h1>Welcome To My Blog</h1>
```
Comment (imhaeunk) [inline index.html]: h2 태그로 바꿔주세요.

**Review by imhaeunk — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (97baek) [reply index.html]: I changed :D

File: index.html (hunk 3 above)
Comment (97baek) [reply index.html]: 수정했습니다!

**Review by 97baek — COMMENTED** [review]
thank you

**Review by imhaeunk — APPROVED** [review]

**Pull request merged**